.github/workflows/ci.yaml

```diff
@@ -156,13 +160,23 @@ jobs:
           # in PR synch the latest commit of the branch is found in github.event.pull_request.head.sha instead
 
           if [ "${{ github.event_name }}" = "release" ] && [ "${{ github.event.action }}" = "released" ]; then
-            echo "::set-output name=tag::${REF_ID/refs-tags-/}"
+            COMMIT_SHA=""
+            IMAGE_TAG="${REF_ID/refs-tags-/}"
           elif [ "${{ github.event_name }}" = "pull_request" ]; then
-            echo "::set-output name=tag::$REF_ID-${{ github.event.pull_request.head.sha }}"
+            COMMIT_SHA="${{ github.event.pull_request.head.sha }}"
+            IMAGE_TAG="$REF_ID-${{ github.event.pull_request.head.sha }}"
```

Comment:
can you make this `IMAGE_TAG="$REF_ID-$COMMIT_SHA"`?

Reply:
sure